mgnify_pipelines_toolkit/analysis/assembly/summarise_goslims.py

```diff
@@ -0,0 +1,198 @@
+import argparse
+import os
+import re
+
+from mgnify_pipelines_toolkit.analysis.assembly.go_utils import parse_ips_file
+
+description = "Go slim pipeline."
```

Comment:
I would move the argparse stuff into the main method.. same reason as for generate_gaf